### PR TITLE
feat: new CDN integration

### PR DESCRIPTION
**Pull request by MartinCupela**
### 🎯 Goal

Refactor and fix proposals for #1752

### CODE REVIEW

**Comment by szuperaz** [comment]
@MartinCupela do you know why there are so many changes here? Most of these isn't related to the CDN integration

**Comment by MartinCupela** [comment]
> @MartinCupela do you know why there are so many changes here? Most of these aren't related to the CDN integration

@szuperaz I apologize, this was due to rebase. Now it is fixed.

**Comment by szuperaz** [comment]
@MartinCupela added this commit: https://github.com/GetStream/stream-chat-react/pull/1773/commits/0768a8ad990706a53fc4d36e09341f4769d42b68

**Comment by szuperaz** [comment]
> > @MartinCupela do you know why there are so many changes here? Most of these aren't related to the CDN integration
> 
> @szuperaz I apologize, this was due to rebase. Now it is fixed.

Thanks

File: src/components/Attachment/AttachmentContainer.tsx [hunk 1]
```diff
@@ -103,16 +103,16 @@ export const GalleryContainer = <
...
-  const configurations = attachment.images.map((image, i) => ({
+  const images = attachment.images.map((image, i) => ({
     ...image,
     previewUrl: attachmentConfigurations[i]?.url,
   }));
 
   return (
     <AttachmentWithinContainer attachment={attachment} componentType='gallery'>
-      <Gallery images={configurations || []} innerRefs={imageElements} key='gallery' />
+      <Gallery images={images || []} innerRefs={imageElements} key='gallery' />
```
Comment (arnautov-anton) [inline src/components/Attachment/AttachmentContainer.tsx:115]:
```suggestion
      <Gallery images={images || []} innerRefs={imageElements} />
```

**Review by szuperaz — DISMISSED** [review]

File: examples/typescript/package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@stream-io/stream-chat-css": "link:../../node_modules/stream-io/stream-chat-css",
+    "@stream-io/stream-chat-css": "link:../../node_modules/stream-io/@stream-chat-css",
```
Comment (MartinCupela) [inline examples/typescript/package.json:6]:
```suggestion
    "@stream-io/stream-chat-css": "link:../../node_modules/@stream-io/stream-chat-css",
```

**Review by petyosi — DISMISSED** [review]

**Review by szuperaz — APPROVED** [review]

**Pull request merged**